_templates/rc-app/new/app.json.ejs.t

```diff
@@ -0,0 +1,19 @@
+---
+to: apps/<%= name %>/app.json
+---
+
+{
+	"id": "",
+	"version": "0.0.1",
+	"requiredApiVersion": "^1.4.0",
```

Comment:
If you're going to use `"@rocket.chat/apps-engine": "^1.19.0",` below in the `package.json`, then this should declare that version.